ethexe/ethereum/src/router/mod.rs

```diff
@@ -198,8 +203,29 @@ impl Router {
                 .map(|signature| Bytes::copy_from_slice(signature.as_ref()))
                 .collect(),
         );
+
+        let mut state_diff = HashMap::default();
```

Comment:
code duplication because the `let builder` type is quite complex and writing generic function would require lot of generics